db/migrate/20200924150827_umm_c_15_3_to_15_4.rb

```diff
@@ -0,0 +1,19 @@
+class UmmC153To154 < ActiveRecord::Migration[5.2]
+  def up
+  end
+
+  def down
+    collection_drafts = CollectionDraft.where.not(draft: {})
+    proposals = CollectionDraftProposal.where.not(draft: {})
+    templates = CollectionTemplate.where.not(draft: {})
+
+    records = collection_drafts + proposals + templates
+
+    records.each do |record|
+      draft = record.draft
+      record['TilingIdentificationSystems'].each do |tiling_id_system|
+        tiling_id_system['TilingIdentificationSystemName'].delete if tiling_id_system['TilingIdentificationSystemName'] = 'Military Grid Reference System'
+      end if record['TilingIdentificationSystems'].present?
+    end
```

Comment:
Need to save the record for the changes to stick, this modifies the data and then discards the modifications.  Should write a test to prove the migration works.